src/Matterhorn/Config.hs

```diff
@@ -375,9 +378,19 @@ getConfig fp = do
                 Just (TokenString _) -> error $ "BUG: getConfig: token in the Config was already a TokenString"
                 Nothing -> return Nothing
 
+            actualOTPToken <- case configOTPToken conf of
+                Just (OTPTokenCommand cmdString) -> do
+                    let (cmd:rest) = T.unpack <$> T.words cmdString
```

Comment:
This will crash if `cmdString` has less than two words. Can you check for that case, please?

Reply:
Sure. To be honest, I've copied that check from `actualToken` to fix that. Looks like there's a bit of code duplication in the branches for actualPass, actualToken and actualOTPToken - is this going to be a problem?

Reply:
No, I can clean that up.